transaction_test.go

```diff
@@ -619,15 +619,10 @@ func TestIteratorAllVersionsWithDeleted(t *testing.T) {
 		require.NoError(t, err)
 
 		// Delete the specific key version from underlying db directly
-		err = db.View(func(txn *Txn) error {
+		err = db.Update(func(txn *Txn) error {
 			item, err := txn.Get([]byte("answer1"))
 			require.NoError(t, err)
-			err = txn.db.batchSet([]*Entry{
```

Comment:
This is an unnormal operation, it overwrites a value with the same version.
We don't need to support it.